src/App.vue

```diff
@@ -1,55 +1,492 @@
+  resize: none;
+  overflow: hidden
+}
+.tweet-bottun-timeline{
+  /* position:absolute; */
+  font-size:5px;
+  color:white;
+  text-align:center;
+  border:solid;
+  border-color:#1D9BF0;
+  background:#1D9BF0;
+  border-radius: 80px;
+  /* margin-left:520px; */
+  padding-top:5px;
+  padding-bottom:5px;
+  width: 70px;
+  margin-top: 10px; 
+}
+.trends{
+  background-color: black;
+  height: 150vh;
+  width: 400px;  
+  overflow: scroll;
+}
+.menu-items{
+  display: flex;
+  margin:0 25.94px ;
+  color:white;
+}
+.home-icon{
+  margin-right: 21px;
+}
+.twitter-icon{
+  margin: 19.75px  25.94px; 
+}
+.tweet-bottun{
+  position:absolute;
+  color:white;
+  text-align:center;
+  border:solid;
+  border-color:#1D9BF0;
+  background:#1D9BF0;
+  border-radius: 80px;
+  margin-left:25px;
+  margin-top:10px;
+  padding-top:15px;
+  padding-bottom:15px;
+  width: 225px;
+}
+.editor-draft{
+  display: flex;
+}
+.editor{
+  padding:12px 16px;
+  border-top: solid 1px gray;
+  border-bottom: solid 1px gray;
+}
+.editor-control{
+  display: flex;
+  justify-content: flex-end;
+}
+.twitter-search{
+  position:absolute;
+  color:#6E767D;
+  text-align:center;
+  border:solid;
+  border-color:#202327;
+  background-color: #202327;
+  border-radius: 80px;
+  margin-left:25px;
+  margin-top:10px;
+  padding-top:10px;
+  padding-bottom:10px;
+  width: 320px;
+}
+.trends-for-you{
+  color:#D9D9D9;
+  border:solid;
+  border-color:#202327;
+  border-radius: 25px;
+  margin-left:25px;
+  margin-top:70px;
+  padding-top:10px;
+  padding-bottom:10px;
+  padding-left: 20px;
+  padding-right: 20px;
+  width: 320px;
+  font-weight: bold;
+  background-color:#202327;
+}
+.who-to-follow{
+  color:#D9D9D9;
+  border:solid;
+  border-color:#202327;
+  border-radius: 25px;
+  margin-left:25px;
+  margin-top:20px;
+  padding-top:10px;
+  padding-bottom:10px;
+  padding-left: 20px;
+  padding-right: 20px;
+  width: 320px;
+  font-weight: bold;
+  background-color:#202327;
+}
+.tweet-container{
+  display:flex;
+  min-height:113px;
+  padding:12px 16px;
+  border-bottom:solid 1px gray;
+}
+.bird{
+  margin-right:12px;
+}
+.user-info{
+  display:flex;
+  font-size:15px;
+}
+.user-name{
+  margin-right:3.25px;
+  font-weight:700;
+}
+.authorize-mark{
+  margin-right:3.25px;
+}
+.tweet-date{
+  color:gray;
+}
+.action-bottun{
+  display:flex;
+  margin-top: 20px;
+  color: gray;
+}
+.reply{
+  margin-right: 20px;
+}
+.reply-number{
+  margin-right: 40px;
+}
+.favorite{
+  margin-right: 20px;
+}
+.favorite-number{
+  margin-right: 40px;
+}
+.retweet{
+  margin-right: 20px;
+}
+.retweet-number{
+  margin-right: 40px;
+}
+.trends-line1{
+  font-size: 13px;
+  color: gray;
+  display: flex;
+  justify-content: space-between;
+}
+.trends-line2{
+  font-size: 15px;
+}
+.trends-line3{
+  font-size: 13px;
+  color: gray;
+}
+.trends-1{
+  margin-top: 20px;
+}
+.trends-2{
+  margin-top: 20px;
+}
+.trends-3{
+  margin-top: 20px;
+}
+.trends-4{
+  margin-top: 20px;
+}
```

Comment:
```suggestion
.trends {
  margin-top: 20px;
}
```

番号いらないよ。クラスは同様の要素に一つ当てるもので
同じスタイル当てるのに番号当てなくていいよ